openconf.cfg

```diff
@@ -31,4 +31,4 @@ source libs/openconf.cfg
 source drivers/openconf.cfg
 source emulators/openconf.cfg
 source tools/openconf.cfg
-
+source ../poc/tcmod/openconf.cfg
```

Comment:
should this path be relative to TCDIR?

Reply:
Yes, we need to adjust this path based on the location of the file.

Reply:
That would work. Will try and update this PR

Reply:
updated the PR